README.md

```diff
@@ -4,6 +4,6 @@ This plugin lets you trigger new builds when your build has completed, with vari
 
 See the documentation and release notes at [Parameterized Trigger Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Parameterized+Trigger+Plugin) on the Jenkins Wiki for more information.
 
-
-
-
+Note: Set Java property "ptp.disableActionViews" (typically Launch Jenkins with -Dptp.disableActionViews) to stop the listing of
```

Comment:
Isn't convention that the property starts with the package id, i.e. in this case hudson.plugins.parameterizedtrigger.disableActionViews?

Reply:
Yes, in Jenkins core it is, but plugins generally seems to want to keep it shorter (like YABV).

Check out the results here: https://github.com/search?q=org%3Ajenkinsci+System.getProperty+repo%3Aplugin&type=Code

Checking the first few pages, the most common way to define system properties for plugins seems to be the usage of a short plugin alias + "." + a verb or noun.

With that in mind, I'm certainly fine with using a package name if that is still what you prefer.